spec/system/admin/customers_spec.rb

```diff
@@ -81,15 +81,15 @@
         expect(page).to have_no_content customer1.email
 
         # Deleting
-        create(:order, customer: customer1)
+        create(:subscription, customer: customer1)
```

Comment:
I know we told you to split your PRs in more commits but I would always combine a spec change with the code change that makes it necessary. Ideally, no commit in the history would fail the specs. That makes it much easier to use `git bisect` to find the cause of a bug.

Reply:
If you want to split it, you can first commit a spec change with the `pending` flag to describe the new expected behaviour. And then you remove the `pending` flag when updating the code.

Reply:
Thanks for the pointer. Will keep that in mind